### ISSUE TITLE
my test_warning might not work on matlab

### ISSUE THREAD

**Issue by cbm755** [catch22/octave-doctest#64]

**Comment by catch22** [comment]
I can reproduce this:

~~~
   >> sprintf('%s', 65)

      expected: warning: implicit conversion from scalar to sq_string ans = A
      got     : ans = A
~~~

**Comment by catch22** [comment]
There is an amusing build failure :smile: 

https://travis-ci.org/catch22/octave-doctest/builds/62947513

**Comment by catch22** [comment]
Closed (0ba76cb9860a5bb091685ebce340dee40439115d).

**Comment by cbm755** [comment]
thank you!
